imap/mboxlist.c

```diff
@@ -5134,13 +5134,12 @@ EXPORTED int mboxlist_checksub(const char *name, const char *userid)
  */
 EXPORTED int mboxlist_changesub(const char *name, const char *userid,
```

Comment:
I know it's ungrateful work, but this function signature could really serve getting its bool arguments refactored in a single `flags` argument.

Reply:
yes, that seems reasonable  - I will do that

Reply:
Hrm, that would change callers like this:

old:
```
mboxlist_changesub(oldname, userid, auth_state, 0, 0, 0, silent);
mboxlist_changesub(newname, userid, auth_state, 1, 0, 0, silent);
mboxlist_changesub(mbentry->name, req->userid, httpd_authstate, 0, 1, 0, 1);
```

new:
```
mboxlist_changesub(oldname, userid, auth_state, silent ? CHANGESUB_SILENT : 0);
mboxlist_changesub(newname, userid, auth_state, CHANGESUB_ADD | (silent ? CHANGESUB_SILENT : 0));
mboxlist_changesub(newname, userid, auth_state, CHANGESUB_FORCE | CHANGESUB_SILENT);

I'm not sure that's better tbh.

```

Reply:
yeah, let's just leave as-is then

Reply:
I left this unchanged, I think it's not worth making that change at this point